_posts/admin/2020-04-17-enable-mptv2.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2020-04-17
+title: Enabling Managed Pipeline Templates
```

Comment:
Enabling Managed Pipeline Templates UI

@fernando-armory can confirm - the feature should still work through spin-CLI - only the UI is being disabled through the flag